package.xml

```diff
@@ -5,7 +5,11 @@
   <author email="isucan@google.com">Ioan Sucan</author>
   <author email="sachinc@sri.com">Sachin Chitta</author>
 
-  <maintainer email="isucan@google.com">Ioan Sucan</maintainer>
+  <maintainer email="dave@dav.ee">Dave Coleman</maintainer>maintainer>
+  <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>maintainer>
+  <maintainer email="me@v4hn.de">Michael Goerner</maintainer>maintainer>
+  <maintainer email="rhaschke@techfak.uni-bielefeld.de">Robert Haschke</maintainer>maintainer>
+  <maintainer email="130s@2000.jukuin.keio.ac.jp">Isaac I. Y. Saito</maintainer>maintainer>
```

Comment:
On Mon, Aug 22, 2016 at 12:27:23AM -0700, Dave Coleman wrote:
> The maintainer tag is malformed!

Crap. Fixed in #28. Please merge it there.